src/mixins/contractMixin.js

```diff
@@ -145,9 +145,8 @@ export const contractMixin = {
             message: 'Ce changement impacte une paie déjà effectuée. Vérifiez que vous ne pouvez pas créer un avenant prenant effet ce mois-ci. Confirmez-vous ce changement ?',
             ok: true,
             cancel: 'Annuler',
-          }).onOk(async () => {
-            await this.saveVersion();
-          }).onCancel(() => NotifyPositive('Edition annulée'));
+          }).onOk(this.saveVersion)
```

Comment:
Si on a besoin du async, ici aussi non ?

Reply:
et la du coup comme on ne fait pas une arrow-function je pense que ce n'est pas necessaire. @jctrebalag tu confirmes ?

Reply:
je confirme 👍 la callback est défini par `this.saveVersion`